frontend/apps/erd-web/next.config.ts

```diff
@@ -2,6 +2,7 @@ import type { NextConfig } from 'next'
 
 const nextConfig: NextConfig = {
   /* config options here */
+  transpilePackages: ['@liam-hq/db-structure', '@liam-hq/erd-core'],
```

Comment:
Is this necessary after all?

Reply:
Not necessary 👍
290ce475503a87acbcb275a66e75b0298a9d3c68